.buildkite/misc/gce-cleanup.yml

```
@@ -0,0 +1,36 @@
+---
+version: "1.0"
+
+accounts:
+  - name: "elastic-platform-ingest"
+    driver: "gce"
+    options:
+      key: "${ACCOUNT_KEY}"
+      secret: "${ACCOUNT_SECRET}"
+      project: "${ACCOUNT_PROJECT}"
+
+scanners:
+  - account_name: "${ACCOUNT_PROJECT}"
```

```suggestion
  - account_name: "elastic-platform-ingest"
```
The account name should match the account name defined at L5.